contributor_path.rst

```diff
@@ -0,0 +1,212 @@
+
+Project-specific guidelines
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+  You don't have to read the documentation mentioned below from cover to cover and memorize everything. It would be good to have general understanding of the workflow and what the documentation consists of to find information you need later when needed.
+
+If you feel confident with the fundamentals above, you should also be familiar with the guidelines specific to the project you chose.
+
+For example, for Ansible Core it is `Ansible development guidelines <https://docs.ansible.com/ansible/latest/dev_guide/index.html>`_.
+
+If you'd like to develop a collection, in addition to it, you should be also familiar with the `Ansible collection development guidelines <https://docs.ansible.com/ansible/latest/dev_guide/developing_collections.html>`_ and development documentation for a particular collection (references can usually be found in the ``README`` or ``CONTRIBUTING`` files in the `collection's repository <https://github.com/ansible-collections/>`_).
+
+First contribution
+==================
+
+Do you first contribution.
+
+Again, you can find some ideas how you can contribute in the `Contribution guide <contribution_to_project.rst>` and the ``README`` / ``CONTRIBUTING`` files of `corresponding repositories <https://github.com/ansible-collections/>`_.
+
+If the contribution you'd like to give implies technical skills, there can be also quick-start guides which can help.
+
+For example, for collections, you can use the `Quick-start development guide <create_pr_quick_start_guide.rst>`_ to learn how to set up everything you need quickly, test your changes, and submit a pull request.
+
+To make your first experience as smooth as possible, read the repository documentation carefully, then ask the repository maintainers for guidance if you have any questions.
+
+Also to look for issues labeled with the ``easyfix``, ``good_first_issue``, and ``docs`` labels can help find good issues to start with.
+
+Continue to contribute
+======================
+
+As was said, when you contribute regularly, your proficiency and judgement in the related area are increasing fast and, along with this, the importance of your presence in the project.
+
+Communicate
+===========
+
+Join the channels
+-----------------
+
+Interact and share your ideas with other folks from the community following `Ansible Code of Conduct <https://docs.ansible.com/ansible/latest/community/code_of_conduct.html>`_.
+
+You can find available communication channels in the `Communication guide <https://docs.ansible.com/ansible/devel/community/communication.html>`_.
+
+The most interactive one is on Libera.Chat IRC where many Ansible projects and working groups have dedicated `channels <https://docs.ansible.com/ansible/devel/community/communication.html#irc-channels>`.
+
+Weekly community IRC meetings
+-----------------------------
```

Comment:
It would be a good opportunity to include the core and docs meeting here as well.

Reply:
Agreed! Added, thanks!